sdk/src/asset_handlers/sfnt_io.rs

```diff
@@ -1129,23 +1145,27 @@ impl CAIWriter for SfntIO {
         input_stream: &mut dyn CAIRead,
         output_stream: &mut dyn CAIReadWrite,
         store_bytes: &[u8],
-    ) -> Result<()> {
-        add_c2pa_to_stream(input_stream, output_stream, store_bytes)
+    ) -> crate::error::Result<()> {
```

Comment:
So I think I prefer the `.map_err(wrap_font_err)` way of mapping the error here, the `Ok()` seems a little too tricky